aleph-client/src/session.rs

```diff
@@ -137,3 +138,22 @@ pub fn wait_for_at_least<C: AnyConnection>(
 pub fn get_session_period<C: AnyConnection>(connection: &C) -> u32 {
     connection.read_constant("Elections", "SessionPeriod")
 }
+
+pub fn get_authorities_for_session<C: AnyConnection>(
+    connection: &C,
+    session: u32,
+    session_period: u32,
```

Comment:
Suggestion: I would remove this parameter.

Reply:
Makes sense, done.